### PR TITLE
wormhole: add page

### PR DESCRIPTION
**Pull request by glmdev**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/wormhole.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+`wormhole send`
+
+- Send a file/directory:
+
+`wormhole send path/to/file`
+
+- Receive a file/directory/text message:
+
+`wormhole receive access_code
```
Comment (sbrl) [inline pages/common/wormhole.md]: Looks like you're missing the closing backtick here :-)

File: pages/common/wormhole.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# wormhole
+
+> Share files from one computer to another, safely.
```
Comment (agnivade) [inline pages/common/wormhole.md]: How about "Share files safely from one compute to another."

File: pages/common/wormhole.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Send a text message:
+
+`wormhole send`
+
+- Send a file/directory:
+
+`wormhole send path/to/file`
```
Comment (agnivade) [inline pages/common/wormhole.md]: {{path/to/file_or_directory}}

File: pages/common/wormhole.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Receive a file/directory/text message:
```
Comment (agnivade) [inline pages/common/wormhole.md]: I don't understand this. If we can send only a file or directory, how can we get a text message also ?

File: pages/common/wormhole.md (hunk 2 above)
Comment (sbrl) [reply pages/common/wormhole.md]: *compute -> computer?

Comment (agnivade) [reply pages/common/wormhole.md]: woops, yeah.

File: pages/common/wormhole.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+`wormhole receive access_code
+
+- Receive SSH keys:
+
+`wormhole ssh invite`
+
+- Send SSH keys:
+
+`wormhole ssh accept access_code`
```
Comment (schneiderl) [inline pages/common/wormhole.md]: I believe the access_code here is a parameter. If so, it should be surrounded by two curly brackets.
{{access_code}}

**Pull request closed without merging**